scripts/covid_tracking_project/historic_us_data/preprocess_csv.py

```diff
@@ -0,0 +1,57 @@
+import csv
+import io
+import urllib.request
+
+output_columns = ['Date',
+                  'CumulativeCount_MedicalTest_COVID_19',
+                  'CumulativeCount_MedicalTest_COVID_19_Positive',
+                  'CumulativeCount_MedicalTest_COVID_19_Negative',
+                  'Count_MedicalTest_COVID_19_Pending',
+                  'CumulativeCount_MedicalConditionIncident_COVID_19_PatientRecovered',
+                  'CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased',
+                  'Count_MedicalConditionIncident_COVID_19_PatientHospitalized',
+                  'CumulativeCount_MedicalConditionIncident_COVID_19_PatientHospitalized',
+                  'Count_MedicalConditionIncident_COVID_19_PatientInICU',
+                  'CumulativeCount_MedicalConditionIncident_COVID_19_PatientInICU',
+                  'Count_MedicalConditionIncident_COVID_19_PatientOnVentilator',
+                  'CumulativeCount_MedicalConditionIncident_COVID_19_PatientOnVentilator'
+                 ]
+with open('COVIDTracking_US.csv', 'w', newline='') as f_out:
```

Comment:
I have an idea. For a subsequent PR, could we modularize State and US code, such that:

- During CSV cleaning phase, add a column 'GeoId' for US where all values are "country/USA"

- A catch is the typeOf for E0 is hardcoded as State. We can table this discussion for now, just something to mull over if you want.

Reply:
Yep! Will have a future version to combine these.